packages/maskbook/src/plugins/ITO/UI/ITO.tsx

```diff
@@ -612,6 +622,10 @@ export function ITO(props: ITO_Props) {
                         className={classes.actionButton}>
                         {t('plugin_ito_withdraw')}
                     </ActionButton>
+                ) : !ifQualified ? (
```

Comment:
Put this button side by side with enter swap button. Assume someone swapped MASK, then spend them all, he/she should still be able to claim.

Reply:
```suggestion
) : !ifQualified && !isBuyer ? (
```

Reply:
Yes, it should not verify the qualification of claimers.